vagrant/provision.sh

```diff
@@ -24,3 +24,44 @@ curl -LJ "https://golang.org/dl/go${GO_VERSION}.linux-amd64.tar.gz" -o go.linux-
 
 # Install ansible
 pip3 install ansible
+
+# Add puppet hosts
+tee -a /etc/hosts << END
+192.168.79.13 puppetserver.example.com puppet
+192.168.79.13 agent.example.com
```

Comment:
What's the purpose of those?

Reply:
if it's essential, could we use `127.0.0.1`?

Reply:
Puppet agent and server communicate using hosts names, certificates for agent and server needs to be generated for different different host names to have the same behaviour as in communication with agent on different machine (in vagrant there is server and agent on the single host). 
I changed IP address to `127.0.0.1`.

Reply:
I suppose using `localhost` and `sumologic-otel-collector` hostnames doesn't work?

Reply:
I changed puppet server to sumologic-otel-collector